python_modules/libraries/dagster-aws/dagster_aws/s3/utils.py

```diff
@@ -28,20 +28,35 @@ def __call__(self, bytes_amount):
 
 
 def construct_s3_client(
-    max_attempts, region_name=None, endpoint_url=None, use_unsigned_session=False, profile_name=None
+    max_attempts,
+    region_name=None,
+    endpoint_url=None,
+    use_unsigned_session=False,
+    profile_name=None,
+    use_ssl=True,
+    verify=None,
+    aws_access_key_id=None,
+    aws_secret_access_key=None,
+    aws_session_token=None,
 ):
     check.int_param(max_attempts, "max_attempts")
     check.opt_str_param(region_name, "region_name")
     check.opt_str_param(endpoint_url, "endpoint_url")
     check.bool_param(use_unsigned_session, "use_unsigned_session")
     check.opt_str_param(profile_name, "profile_name")
+    check.bool_param(use_ssl, "use_ssl")
+    check.bool_param(verify, "verify")
```

Comment:
probably want to check parameterization on last three args here as well

Reply:
good catch. Sorry I forgot that